src/js/builtins/JSBufferPrototype.ts

```diff
@@ -427,29 +427,29 @@ export function hexWrite(this: BufferExt, text, offset, length) {
   return this.write(text, offset, length, "hex");
 }
 
-export function utf8Slice(this: BufferExt, offset, length) {
-  return this.toString(offset, length, "utf8");
+export function utf8Slice(this: BufferExt, start, end) {
+  return this.toString("utf8", start, end);
```

Comment:
`start` and `end` are used instead of `offset` and `length` to maintain compatibility with nodejs.